inc/sainexthopgroup.h

```diff
@@ -184,6 +184,41 @@ typedef enum _sai_next_hop_group_attr_t
      */
     SAI_NEXT_HOP_GROUP_ATTR_SELECTION_MAP,
 
+    /**
+     * @brief Adaptive routing and switching object for this group.
+     *
+     * @type sai_object_id_t
+     * @flags CREATE_AND_SET
```

Comment:
Should it be CREATE_ONLY or you allow to bind the ARS object to any non-empty NHG  with CREATE_AND_SET ?

Reply:
It can be assigned or unassigned anytime in a running system hence CREATE_AND_SET.